### PR TITLE
extendrtests: added null externalptr test

### PR DESCRIPTION
**Pull request by CGMossa**
Follow-up on #709, as we didn't actually add a test for this in `tests/extendrtests`.

### CODE REVIEW

**Comment by eitsupi** [comment]
I'm confused, #709 is still open, but if this test succeeds, does that mean it's already fixed?

**Comment by CGMossa** [comment]
Yeah, it's already fixed.
But the test shows that it's fixed, thus it can close the issue.

File: tests/extendrtests/tests/testthat/test-classes.R [hunk 1]
```diff
@@ -30,3 +30,13 @@ test_that("Issue 431: Restore struct as ExternalPtr", {
...
+
+test_that("Error nicely with null externalptr", {
+  x <- MyClass$new()
+  x$set_a(42L)
+  null_x <- new("externalptr") # no attributes, null-ptr
+  attributes(null_x) <- attributes(x)
+  expect_error(
+    null_x$set_a(44L)
+  )
+})
```
Comment (Ilia-Kosenkov) [inline tests/extendrtests/tests/testthat/test-classes.R:42]: R files like new lines at the end.
```suggestion
})

```

**Review by Ilia-Kosenkov — APPROVED** [review]
LGTM

**Pull request merged**